src/ui/components/Icon/styles.scss

```diff
@@ -1,20 +1,24 @@
 @import "./vars";
 @import "~core/css/inc/mixins";
 
-.Icon {
+@mixin icon($name) {
+  background: url("./#{$name}.svg") center no-repeat;
+  background-size: contain;
   content: "";
   display: inline-block;
   height: $default-icon-size;
   width: $default-icon-size;
 }
 
 .Icon-magnifying-glass {
-  background: url("~amo/img/icons/magnifying-glass.svg") center no-repeat;
```

Comment:
Sigh. I see you found some cobwebs. I don't see the file path to `magnifying-glass.svg` used directly so I think moving it was safe. Thanks.